pkg/client/status_reporter.go

```diff
@@ -24,6 +24,7 @@ import (
 	clientv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	gostr "strings"
```

Comment:
Would it be possible to do it the other way round and remap "github.com/openshift/cluster-monitoring-operator/pkg/strings" import instead of default golang one? IMHO it should help with code readability. WDYT?

Reply:
Done